build/build-bundle.js

```diff
@@ -91,6 +91,7 @@ async function build(entryPath, distPath, opts = {minify: true}) {
   const shimsObj = {};
 
   const modulesToIgnore = [
+    'puppeteer',
```

Comment:
Puppeteer isn't needed if the environment provides a puppeteer page as a dependency. We don't need puppeteer bloating our bundle.